kcauto.sikuli/util.sikuli/util.py

```diff
@@ -474,6 +474,7 @@ def click_mouse(cls, region):
         region.mouseDown(Button.LEFT)
         cls.kc_sleep()
         region.mouseUp(Button.LEFT)
+        cls.log_msg("Click")
```

Comment:
If we're adding a console log for the click, let's make it more informative. [Env.getMouseLocation](http://doc.sikuli.org/globals.html#Env.getMouseLocation) returns a Sikuli `Location` object where we can get the `x` and `y` location of the mouse. If we add this information to the console log it'll make things easier to debug.

Reply:
Will try.